testing/steps/test_logic.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+
+from neuraxle.steps.logic import FilterColumns, Greater, Less, If
+
+
+def test_filter():
+    data_inputs = np.random.random((10, 2))
+    filter = FilterColumns([
+        (0, Greater(then=0.5)),
+        (1, Less(then=0.5))
+    ])
+
+    outputs = filter.transform(data_inputs)
+
+    for di in outputs:
+        assert di[0] > 0.5
+        assert di[1] < 0.5
```

Comment:
I really don't understand what are those asserts and why we don't compare on 10 items or something else like that... Did it transpose the array or something?